packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -584,12 +581,13 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
 
         await this.prepareVariantOptions(variantOp, product.id)
 
+        const updateData = transformVariantData(variantOp)
+        delete updateData.product
```

Comment:
question: What led to this change? Just curious, as it seems unrelated to the PR scope

Reply:
It is unrelated, but I encounter this one as it is not expected by the update method, product should not exists, otherwise if a cascade occurs it might update the product where you don't expect it to be